sql/core/src/test/resources/sql-tests/inputs/group-by.sql

```diff
@@ -53,3 +53,13 @@ set spark.sql.groupByAliases=false;
 
 -- Check analysis exceptions
 SELECT a AS k, COUNT(b) FROM testData GROUP BY k;
+
+-- turn on group by aliases (for remaining tests)
+set spark.sql.groupByAliases=true;
+
+-- Aggregate with empty input and non-empty GroupBy expressions.
+SELECT a, COUNT(1) FROM testData WHERE false GROUP BY a;
+
+-- Aggregate with empty input and empty GroupBy expressions.
+SELECT COUNT(1) FROM testData WHERE false;
```

Comment:
This test case would have passed before because it contains aggregate expressions. The test case on the next line is the one which actually failed because column pruning eliminates the aggregate expression, leaving us with an Aggregate with no aggregate expressions.